content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -152,6 +152,52 @@ kubectl label nodes <node-with-p100> accelerator=nvidia-tesla-p100
 
 ## Automatic node labelling {#node-labeller}
 
+### NVIDIA
+
+The [NVIDIA GPU Feature Discovery](https://github.com/NVIDIA/gpu-feature-discovery) project allows you to automatically generate labels depending on the GPUs available on your nodes. It uses the [Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery) project from Kubernetes to label nodes.
+
+The currently available labels are:
+
+| Label Name                     | Value Type | Meaning                                  |
+| -------------------------------| ---------- | ---------------------------------------- |
+| nvidia.com/cuda.runtime.major  | Integer    | Major of the version of CUDA             |
+| nvidia.com/cuda.runtime.minor  | Integer    | Minor of the version of CUDA             |
+| nvidia.com/cuda.driver.major   | Integer    | Major of the version of NVIDIA driver    |
+| nvidia.com/cuda.driver.minor   | Integer    | Minor of the version of NVIDIA driver    |
+| nvidia.com/cuda.driver.rev     | Integer    | Revision of the version of NVIDIA driver |
+| nvidia.com/gpu.family          | String     | Architecture family of the GPU           |
+| nvidia.com/gpu.machine         | String     | Machine type                             |
+| nvidia.com/gpu.product         | String     | Model of the GPU                         |
+| nvidia.com/gpu.memory          | Integer    | Memory of the GPU in Mb                  |
+| nvidia.com/gpu.compute.major   | Integer    | Major of the compute capabilities        |
+| nvidia.com/gpu.compute.minor   | Integer    | Minor of the compute capabilities        |
+| nvidia.com/gfd.timestamp       | Integer    | Timestamp of the generated labels        |
```

Comment:
I would put the label names in backticks, and sort them.

Reply:
Addressed!